src/components/pagination-nav/pagination-nav.js

```diff
@@ -19,10 +20,7 @@ const NAME = 'BPaginationNav'
 
 // --- Props ---
 
-const linkProps = pluckProps(
-  ['activeClass', 'exact', 'exactActiveClass', 'prefetch', 'noPrefetch', 'routerComponentName'],
-  BLinkProps
-)
+const linkProps = omit(BLinkProps, ['event'])
```

Comment:
maybe omit `routerTag` from this as well. as it would break semantic markup of the pagination component.

Reply:
`routerTag` is something that should only be used for raw `<b-link>` usage.

Reply:
So maybe it needs to be omitted from everything except `<b-link>`